### DISCUSSION TITLE
bci python 3.8 - 3.10

### DISCUSSION THREAD

**Discussion by prabhu** [SUSE/bci#21] [Ideas]
Could you kindly add python 3.8 - 3.10 to the bci images?

**Comment by dcermak** [comment]
Thanks for the suggestion, but unfortunately we cannot. We are only shipping whatever we have available in SUSE Linux Enterprise Server and that is at the moment only python 3.6, 3.11 and 3.12. If you are a SUSE Customer, then you can request additional python versions via the usual channels. Otherwise I can point you to our community version based on openSUSE Tumbleweed:
- registry.opensuse.org/opensuse/bci/python:3.10
- registry.opensuse.org/opensuse/bci/python:3.9 (note that this one is no longer being updated, consider it EoL)

**Reply by prabhu** [reply]
Thank you! I managed to get 3.10 working with tumbleweed image. Good to know there is an official 3.10 version available though.
